zebra/zebra_ns_notify.c

```diff
@@ -0,0 +1,227 @@
+/*
+ * Zebra NS collector and notifier for Network NameSpaces
+ *
+ * Copyright (C) 2017 by 6WIND
+ *
+ * This file is part of FreeRangeRouting (FRR)
```

Comment:
This is the wrong copyright header, please use the one listed in `COMMUNITY.md` (i.e. do not include `FRR` or `This file is part of...`)